apps/src/gamelab/blocks.js

```diff
@@ -1,22 +1,30 @@
 import { appendCategory, createJsWrapperBlockCreator } from '../block_utils';
+import { getStore } from '../redux';
 
 const SPRITE_COLOR = [184, 1.00, 0.74];
 const EVENT_COLOR = [140, 1.00, 0.74];
 const EVENT_LOOP_COLOR = [322, 0.90, 0.95];
 const VARIABLES_COLOR = [312, 0.32, 0.62];
 const WORLD_COLOR = [240, 0.45, 0.65];
 
-const SPRITES = [
-  ['dog', '"dog"'],
-  ['cat', '"cat"'],
-];
-
 export default {
   install(blockly, blockInstallOptions) {
     // TODO(ram): Create Blockly.BlockValueType.SPRITE
     const SPRITE_TYPE = blockly.BlockValueType.NONE;
     const { ORDER_MEMBER } = Blockly.JavaScript;
 
+    const sprites = () => {
+      const animationList = getStore().getState().animationList;
+      if (animationList.orderedKeys.length === 0) {
+        console.warn("No sprites available");
```

Comment:
Does console stuff still break IE when the developer tools are closed?

Reply:
Looks like that was fixed as of IE10, and officially we only support IE11+.

We use `console.log` in enough places that I'm guessing things are already pretty broken IE8/9.